src/networking/spotify.api.ts

```diff
@@ -52,19 +52,19 @@ export class SpotifyApi {
             });
     }
 
-    getRecommendations = (genres?: string[], artists?: string[], retry = false): Promise<Track[]> => {
+    getRecommendations = (limit: number, genres?: string[], artists?: string[], retry = false): Promise<Track[]> => {
```

Comment:
I think it's better to make the new limit param optional, and append it to the end of the list.. you can give it a default value (20)